src/store/storage-controller.ts

```diff
@@ -111,19 +109,35 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public get MessageStore(): MessageStore { return this.messageStore; }
+
+  public queryMessageStore(tenant: string, filter: Filter, options?: MessageStoreOptions): Promise<BaseMessage[]> {
+    return this.messageStore.query(tenant, filter, options);
+  }
+
+  public get(tenant: string, messageCid: string, dataCid: string): Promise<GetResult|undefined> {
```

Comment:
i think `getData` might be more clear since this is just returning the data, not the message (also matches nicely with [src/interfaces/messages/handlers/messages-get.ts:39](https://github.com/TBD54566975/dwn-sdk-js/pull/338/files#diff-5d5c842d6507a8df7c2f87057043b86e17d4fc15a5735e010e2c3c3d79c765a0R39))

Reply:
Addressed in https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/8a39de45aaa23185e406373a304c4ee9e4d68730